ui/src/telegrafs/components/CollectorCard.tsx

```diff
@@ -78,7 +77,9 @@ class CollectorRow extends PureComponent<Props & WithRouterProps> {
         metaData={() => [
           <>Bucket: {bucket}</>,
           <>
-            <a onClick={this.handleOpenInstructions}>Setup Instructions</a>
+            <Link to={this.handleOpenInstructions}>
```

Comment:
to should just be a string, not a function

Reply:
Thanks! I updated it